NitroxClient/GameLogic/Spawning/Metadata/Processor/Abstract/EntityMetadataProcessor.cs

```diff
@@ -4,13 +4,13 @@
 
 namespace NitroxClient.GameLogic.Spawning.Metadata.Processor.Abstract;
 
-public abstract class EntityMetadataProcessor<T> : IEntityMetadataProcessor where T : EntityMetadata
+public abstract class EntityMetadataProcessor<TMetadata> : IEntityMetadataProcessor where TMetadata : EntityMetadata
```

Comment:
is this here to just enforce it having to be of the interface type rather than just anything?

Reply:
There's a constraint that requires TMetadata to be of type EntityMetadata if that's what you mean